server/integration-test/src/test/java/com/oceanbase/odc/service/session/ConnectConsoleServiceTest.java

```diff
@@ -78,21 +83,300 @@
  */
 public class ConnectConsoleServiceTest extends ServiceTestEnv {
 
-    private final String sessionid = "10000";
+    private final String sessionId = "10000";
     @MockBean
     private ConnectSessionService sessionService;
     @Autowired
     private ConnectConsoleService consoleService;
     @Autowired
     private DefaultDBSessionManage defaultConnectSessionManage;
 
+    @Test
+    public void editProcedureForOBMysql_normal_successResult() throws Exception {
+        ConnectionSession testConnectionSession = TestConnectionUtil.getTestConnectionSession(ConnectType.OB_MYSQL);
+        SyncJdbcExecutor syncJdbcExecutor = testConnectionSession.getSyncJdbcExecutor(
+                ConnectionSessionConstants.CONSOLE_DS_KEY);
+        String dropTestProcedure = "DROP PROCEDURE IF EXISTS ODC_TEST_PROCEDURE;";
```

Comment:
should use IsolatedNameGenerator for generate procedureName ,
for avoid pararell conflict between test cases.